contracts/mocks/ConfidentialFungibleTokenERC20WrapperMock.sol

```diff
@@ -1,16 +1,11 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.24;
 
-import {SepoliaZamaFHEVMConfig} from "fhevm/config/ZamaFHEVMConfig.sol";
-import {SepoliaZamaGatewayConfig} from "fhevm/config/ZamaGatewayConfig.sol";
+import {SepoliaConfig} from "@fhevm/solidity/config/ZamaConfig.sol";
 import {ConfidentialFungibleTokenERC20Wrapper, ConfidentialFungibleToken} from "../token/extensions/ConfidentialFungibleTokenERC20Wrapper.sol";
 import {IERC20} from "@openzeppelin/contracts/interfaces/IERC20.sol";
 
-contract ConfidentialFungibleTokenERC20WrapperMock is
-    SepoliaZamaFHEVMConfig,
-    SepoliaZamaGatewayConfig,
-    ConfidentialFungibleTokenERC20Wrapper
-{
+contract ConfidentialFungibleTokenERC20WrapperMock is SepoliaConfig, ConfidentialFungibleTokenERC20Wrapper {
```

Comment:
Maybe we should generally maintain a consistent order inheritance (eg here the ones of `ConfidentialFungibleTokenERC20WrapperMock`, `ConfidentialFungibleTokenMock`)?